README.md

```diff
@@ -126,6 +126,9 @@ The supported commands are divided into groups according to their target, at thi
 * `backup --action [backup|restore|clean]` - This command enables you to use [Kontent.ai backup manager](https://github.com/kontent-ai/backup-manager-js)
   * The purpose of this tool is to backup & restore [Kontent.ai projects](https://kontent.ai/). This project uses CM API to both get & restore data.
 
+* `status --implementation` - This command creates in your working directory a new directory with a file `statusImpl.ts`. Having this file created, CLI will take your custom implementation of saving/reading the migrations status.

```

Comment:
I am not sure about the name of the file.

Reply:
Do we want to add this scaffolding in the implementation as a command? 

I would expect to tell them -> hay there is an expected file with this function with this signature. It should be in js (and if you want to use TS set up a transpilation).